### PR TITLE
Admin :: Categories:: Active

### PR DESCRIPTION
**Pull request by TrucLuu**

### CODE REVIEW

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -41,6 +42,11 @@ def destroy
     redirect_to admin_categories_url
   end
 
+  def activate
+    @category.set_activate(params: params)
+    redirect_to :back
```
Comment (ducminhvu) [inline app/controllers/admin/categories_controller.rb]: remove ()

File: app/controllers/admin/categories_controller.rb [hunk 2]
```diff
@@ -41,6 +42,11 @@ def destroy
     redirect_to admin_categories_url
   end
 
+  def activate
+    @category.set_activate(params: params)
```
Comment (ducminhvu) [inline app/controllers/admin/categories_controller.rb]: remove ()

File: app/controllers/admin/categories_controller.rb [hunk 3]
```diff
@@ -41,6 +42,11 @@ def destroy
     redirect_to admin_categories_url
   end
 
+  def activate
```
Comment (dieunb) [inline app/controllers/admin/categories_controller.rb]: restful?

File: app/views/admin/categories/_category.html.erb [hunk 4]
```diff
@@ -1,6 +1,12 @@
 <tr>
   <td><%= category.name %></td>
   <td><%= category.content %></td>
+  <td>
+    <%= form_for [:admin, category], method: :patch do |f| %>
```
Comment (dieunb) [inline app/views/admin/categories/_category.html.erb]: no need `method: :patch`

**Pull request merged**